plugins/view-transitions/includes/settings.php

```diff
@@ -228,7 +228,7 @@ function plvt_apply_settings_to_theme_support(): void {
 function plvt_add_setting_ui(): void {
 	add_settings_section(
 		'plvt_view_transitions',
-		__( 'View Transitions', 'view-transitions' ),
+		_x( 'View Transitions', 'Setting section in Reading page', 'view-transitions' ),
```

Comment:
Maybe make the comment less specific about the exact location of the section. I don't think the location is relevant for the translation and if it was later moved somewhere else, we wouldn't want to update (and invalidate translations of) this string.
```suggestion
		_x( 'View Transitions', 'Settings section', 'view-transitions' ),
```

Reply:
You are right 👍👍👍👍👍